frontend/src/pages/Account.tsx

```diff
@@ -0,0 +1,280 @@
+                tmpChipData.push({key: i, label: user.roles[i]});
+            }
+            setChipData(tmpChipData);
+        }
+        if (user.wca_person != null) {
+            default_WCAID = user.wca_person;
+        }
+    }
+
+    const ListItem = styled('li')(({theme}) => ({
+        margin: theme.spacing(0.5),
+    }));
+
+
+    const handleSaveProfile = async () => {
+        try {
+            const resp = await httpClient.post(API_BASE_URL + "/edit", {
+                province: province ? province.id : 'na',
+            });
+            if (resp.data.success === true) {
+                setAlertContent(t("account.success"));
+                setAlertType("success");
+                setAlert(true);
+            } else {
+                setAlertContent(t("account.error"));
+                setAlertType("error");
+                setAlert(true);
+            }
+        } catch (error: any) {
+            if (error.response.status === 401) {
+                console.log("Invalid credentials" + error.response.status);
+            } else if (error.response.status === 403) {
+                console.log("Forbidden" + error.response.status);
+            } else if (error.response.status === 404) {
+                console.log("Not found" + error.response.status);
+            }
+        }
+    };
+
+    const handleSignIn = async () => {
+        setLoading(true);
+        signIn();
+    }
+
+    return (
+        <Container maxWidth="md">
+            <Box marginY="4rem">
+                <Typography component="h1" variant="h3" fontWeight="bold" gutterBottom>
+                    {t("account.title")}
+                </Typography>
+            </Box>
+            {loading ? <CircularProgress/>
+                : user != null ? (
+                    <div>
+                        <Box marginY="2rem">
+                            <Typography
+                                component="h2"
+                                variant="h5"
+                                fontWeight="bold"
+                                gutterBottom
+                                marginY="1rem"
+                            >
+                                {t("account.hi")}{user.name}!
+                            </Typography>
+
+                            <Autocomplete
+                                disablePortal
+                                id="combo-box-demo"
+                                options={provinces}
+                                sx={{width: 300}}
+                                value={province || default_province}
+                                defaultValue={default_province}
+                                onChange={(event, newValue) => {
+                                    setProvince(newValue);
+                                    if (newValue == null) {
+                                    } else if (newValue.id === "qc") {
+                                        console.log("Vive le Québec libre!");
```

Comment:
is this going to be kept?

Reply:
I wanted to keep it as a small easter egg, if that's an issue I can remove it